network/server/tchannelthrift/node/service.go

```diff
@@ -277,3 +278,8 @@ func (s *service) WriteBatch(tctx thrift.Context, req *rpc.WriteBatchRequest) er
 	}
 	return nil
 }
+
+func (s *service) TruncateNamespace(tctx thrift.Context, req *rpc.TruncateNamespaceRequest) error {
+	s.db.TruncateNamespace(req.Ns)
+	return nil
```

Comment:
Yeah we should return something from this method describing something about the result of the operation so we can tell if it did things when we run it.